tuneme/settings/base.py

```diff
@@ -180,10 +180,10 @@
     'CELERY_RESULT_BACKEND', 'redis://localhost:6379/0')
 CELERYBEAT_SCHEDULE = {
     # Executes every morning at 8:00 A.M GMT+2
-    'add-every-morning': {
-        'task': 'molo.profiles.task.send_user_data_to_slack',
-        'schedule': crontab(minute=0, hour=8)
-    },
+    # 'add-every-morning': {
+    #     'task': 'molo.profiles.task.send_user_data_to_slack',
+    #     'schedule': crontab(minute=0, hour=8)
+    # },
```

Comment:
Do you want to comment out or delete? I think we're deprecating this feature anyway?